### ISSUE TITLE
console error on player send

### ISSUE THREAD

**Issue by firebl0od** [TheLimeGlass/Skungee#44]
hi, when i try to send a player to an other server nothing happen, and i get an error on the proxy.

my code
```python
on inventory click:
	if {iped.%player%} is false:
		cancel event
		stop
	if {logged.%player%} is false:
		cancel event
		stop
	if inventory name of player's current inventory is "&cserveurs":
		cancel event
		"%event-slot's name%" is "&aThalion":
			skungee send bungee player to "thalion"
		"%event-slot's name%" is "&aSurvie":
			skungee send bungee player to "survie"
		"%event-slot's name%" is "&aCreatif":
			skungee send bungee player to "creatif"
		"%event-slot's name%" is "&aMyona":
			skungee send bungee player to "myöna"
		"%event-slot's name%" is "&aSkywars":
			skungee send bungee player to "skywars"
		"%event-slot's name%" is "&aNearDead":
			skungee send bungee player to "neardead"
		"%event-slot's name%" is "&aSurvie Vanilla":
			skungee send bungee player to "vanilla"
```

and the error (note that le skript reload fine and there is no error on the servers)

```python
18:12:25 [GRAVE] java.lang.NoClassDefFoundError: net/md_5/bungee/api/ServerConnectRequest
18:12:25 [GRAVE] 	at me.limeglass.skungee.bungeecord.handlers.PlayerConnectHandler.handlePacket(PlayerConnectHandler.java:28)
18:12:25 [GRAVE] 	at me.limeglass.skungee.bungeecord.handlercontroller.SkungeeHandler.callPacket(SkungeeHandler.java:98)
18:12:25 [GRAVE] 	at me.limeglass.skungee.bungeecord.sockets.BungeeRunnable.run(BungeeRunnable.java:98)
18:12:25 [GRAVE] 	at java.lang.Thread.run(Thread.java:748)
18:12:25 [GRAVE] Caused by: java.lang.ClassNotFoundException: net.md_5.bungee.api.ServerConnectRequest
18:12:25 [GRAVE] 	at net.md_5.bungee.api.plugin.PluginClassloader.loadClass0(PluginClassloader.java:53)
18:12:25 [GRAVE] 	at net.md_5.bungee.api.plugin.PluginClassloader.loadClass(PluginClassloader.java:27)
18:12:25 [GRAVE] 	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
18:12:25 [GRAVE] 	... 4 more
18:12:26 [GRAVE] Exception in thread "Thread-161" 
18:12:26 [GRAVE] java.lang.NoClassDefFoundError: net/md_5/bungee/api/ServerConnectRequest
18:12:26 [GRAVE] 	at me.limeglass.skungee.bungeecord.handlers.PlayerConnectHandler.handlePacket(PlayerConnectHandler.java:28)
18:12:26 [GRAVE] 	at me.limeglass.skungee.bungeecord.handlercontroller.SkungeeHandler.callPacket(SkungeeHandler.java:98)
18:12:26 [GRAVE] 	at me.limeglass.skungee.bungeecord.sockets.BungeeRunnable.run(BungeeRunnable.java:98)
18:12:26 [GRAVE] 	at java.lang.Thread.run(Thread.java:748)
18:12:26 [GRAVE] Exception in thread "Thread-162" 
18:12:26 [GRAVE] java.lang.NoClassDefFoundError: net/md_5/bungee/api/ServerConnectRequest
18:12:26 [GRAVE] 	at me.limeglass.skungee.bungeecord.handlers.PlayerConnectHandler.handlePacket(PlayerConnectHandler.java:28)
18:12:26 [GRAVE] 	at me.limeglass.skungee.bungeecord.handlercontroller.SkungeeHandler.callPacket(SkungeeHandler.java:98)
18:12:26 [GRAVE] 	at me.limeglass.skungee.bungeecord.sockets.BungeeRunnable.run(BungeeRunnable.java:98)
18:12:26 [GRAVE] 	at java.lang.Thread.run(Thread.java:748)
18:12:26 [GRAVE] Exception in thread "Thread-163" 
18:12:26 [GRAVE] java.lang.NoClassDefFoundError: net/md_5/bungee/api/ServerConnectRequest
18:12:26 [GRAVE] 	at me.limeglass.skungee.bungeecord.handlers.PlayerConnectHandler.handlePacket(PlayerConnectHandler.java:28)
18:12:26 [GRAVE] 	at me.limeglass.skungee.bungeecord.handlercontroller.SkungeeHandler.callPacket(SkungeeHandler.java:98)
18:12:26 [GRAVE] 	at me.limeglass.skungee.bungeecord.sockets.BungeeRunnable.run(BungeeRunnable.java:98)
18:12:26 [GRAVE] 	at java.lang.Thread.run(Thread.java:748)
18:12:27 [GRAVE] Exception in thread "Thread-164" 
18:12:27 [GRAVE] java.lang.NoClassDefFoundError: net/md_5/bungee/api/ServerConnectRequest
18:12:27 [GRAVE] 	at me.limeglass.skungee.bungeecord.handlers.PlayerConnectHandler.handlePacket(PlayerConnectHandler.java:28)
18:12:27 [GRAVE] 	at me.limeglass.skungee.bungeecord.handlercontroller.SkungeeHandler.callPacket(SkungeeHandler.java:98)
18:12:27 [GRAVE] 	at me.limeglass.skungee.bungeecord.sockets.BungeeRunnable.run(BungeeRunnable.java:98)
18:12:27 [GRAVE] 	at java.lang.Thread.run(Thread.java:748)
```

**Comment by firebl0od** [comment]
and i have the last version 1.0.9a build with gradle

**Comment by firebl0od** [comment]
found a way to send player anyway so im fine, it's more for you to know that this doesn't work

where 's what iv done instead
`skungee run bungee proxy command "/send %player% thalion"`

**Comment by TheLimeGlass** [comment]
If you update your Bungeecord to 1.13 it will work. Bungeecord 1.13 allows all client versions. They just call it 1.13 to match the latest. It has nothing to do with the client versions, so I suggest you upgrade.

**Comment by TheLimeGlass** [comment]
Added legacy support for fix in 1.1.0
